popup.css

```
@@ -149,9 +156,18 @@ border-radius: 5px;
   }
 
   .runtime {
-    right: 12%;
+    right: 8%;
     margin-bottom: 5px;
   }
+  .story {
+	  font-size: 15px;
+	  text-align: center;
+	  color: #FFFFFF;
+  }
+  .credits {
+	  font-size: 11px;
+	  color: #ADD8E6;
+  }
```

```suggestion
  .story {
	  font-size: 14px;
  }
  .credits {
	  font-size: 12px;
	  color: #ADD8E6;
  }
```
Credits can be a lighter color.

---

plz, provide the hex of the one u want.

---

Feel free to pick something that doesn't break the accessibility and doesn't look off with rest of the colors. I can update it later if I find a more suitable color.

Also @AMMARmms you can just apply these suggestions from GitHub. You will see `Add suggestion to batch` and then you can commit multiple suggestions at once. Then you can simply pull the new changes to your local and try them out.